custom-words.txt

```diff
@@ -2456,4 +2458,5 @@ testrun
 loadtest
 xvalue
 yvalue
-servermetrics
\ No newline at end of file
+servermetrics
+Obuscated
```

Comment:
I don't believe "Obuscated" is a word.  You probably want "Obfuscated".

Reply:
Good catch! 
I also notice this is not used any more, so I remove this term from the custom-words list.